locales/zh_CN/LC_MESSAGES/snode.po

```diff
@@ -95,15 +107,15 @@ msgstr ""
 
 #: ../../snode.rst:36 ../../snode.rst:65
 msgid "(Tensor)"
-msgstr ""
+msgstr "(张量)"
 
 #: ../../snode.rst:37
 msgid "(tuple of integers) the shape of tensor"
-msgstr ""
+msgstr "(元组或标量) 张量的形状"
 
 #: ../../snode.rst:39
 msgid "For example,"
-msgstr ""
+msgstr "例如,"
```

Comment:
We use cn comma in cn context, see [wiki](https://github.com/taichi-dev/taichi-docs-zh-cn/wiki/%E4%B8%BB%E9%A1%B5):
```suggestion
msgstr "例如，"
```